Bug: 没有推送

### 操作系统

Windows

### Python 版本

3.11.0

### NoneBot 版本

2.0.0

### Bilichat 版本

6.0.11

### 描述问题

没有推送动态和直播通知

### 插件的配置项

```Text

```

### 截图或日志

```Text
~~~
01-18 12:30:13 [ERROR] apscheduler | Job "dynamic (trigger: interval[0:03:00], next run at: 2025-01-18 12:33:21 CST)" raised an exception
Traceback (most recent call last):
  File "<string>", line 15, in <module>
  File "C:\bot\bilichat-bot\.venv\lib\site-packages\nonebot\__init__.py", line 337, in run
    get_driver().run(*args, **kwargs)
  File "C:\bot\bilichat-bot\.venv\lib\site-packages\nonebot\drivers\fastapi.py", line 187, in run
    uvicorn.run(
  File "C:\bot\bilichat-bot\.venv\lib\site-packages\uvicorn\main.py", line 579, in run
    server.run()
  File "C:\bot\bilichat-bot\.venv\lib\site-packages\uvicorn\server.py", line 66, in run
    return asyncio.run(self.serve(sockets=sockets))
  File "C:\Users\admini\AppData\Local\Programs\Python\Python310\lib\asyncio\runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "C:\Users\admini\AppData\Local\Programs\Python\Python310\lib\asyncio\base_events.py", line 636, in run_until_complete
    self.run_forever()
  File "C:\Users\admini\AppData\Local\Programs\Python\Python310\lib\asyncio\windows_events.py", line 321, in run_forever
    super().run_forever()
  File "C:\Users\admini\AppData\Local\Programs\Python\Python310\lib\asyncio\base_events.py", line 603, in run_forever
    self._run_once()
  File "C:\Users\admini\AppData\Local\Programs\Python\Python310\lib\asyncio\base_events.py", line 1909, in _run_once
    handle._run()
  File "C:\Users\admini\AppData\Local\Programs\Python\Python310\lib\asyncio\events.py", line 80, in _run
    self._context.run(self._callback, *self._args)
> File "C:\bot\bilichat-bot\.venv\lib\site-packages\apscheduler\executors\base.py", line 181, in run_coroutine_job
    retval = await job.func(*job.args, **job.kwargs)
  File "C:\bot\bilichat-bot\.venv\lib\site-packages\nonebot_plugin_bilichat\subscribe\fetch_and_push.py", line 37, in dynamic
    up.dyn_offset = max([dyn.dyn_id for dyn in all_dyns])
ValueError: max() arg is an empty sequence
01-18 12:30:19 [INFO] uvicorn | 127.0.0.1:61428 - "POST /bilichatapi/subs/lives HTTP/1.1" 200
~~~
```

--- a/nonebot_plugin_bilichat/subscribe/fetch_and_push.py
+++ b/nonebot_plugin_bilichat/subscribe/fetch_and_push.py
@@ -29,6 +29,9 @@ async def dynamic():
         api = get_request_api()
         try:
             all_dyns = await api.subs_dynamic(up.uid)
+            if not all_dyns:
+                logger.info(f"[Dynamic] UP {up.name}({up.uid}) 未发布动态")
+                continue
         except Exception as e:
             logger.error(f"[Dynamic] 获取 UP {up.name}({up.uid}) 动态失败: {e}")
             continue

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,7 +1,7 @@
 [project]
 name = "nonebot-plugin-bilichat"
 
-version = "6.0.11"
+version = "6.0.12"
 
 description = "全功能的 bilibili 内容解析器及订阅器"
 authors = [

